### ISSUE TITLE
Is it dead?

### ISSUE THREAD

**Issue by nosachamos** [javascriptdata/danfojs#648]
Is the lib still being maintained? It looks so promising, but with so many pending PRs and open issues I'm wondering if the project is abandoned.

**Comment by GauravB159** [comment]
Seems pretty abandoned. Does anyone know a comparable library? This works for my use case but there are quite a few issues I need to work around.

**Comment by nosachamos** [comment]
Yes, I have settled for pyodide instead. It's a heavier solution but you
get to use pandas directly

**Comment by cgjosephlee** [comment]
Try polars:
- https://github.com/pola-rs/nodejs-polars
- https://github.com/pola-rs/js-polars

**Comment by risenW** [comment]
@nosachamos No it's not, we just need more maintainers. At the moment, I am the sole maintainer of this package, and I need to balance that with personal stuffs as well. That being said, I have started making some new updates and actively reviewing old PRs
